apps/marketplace/components/Brief/Overview/BriefOverview.js

```diff
@@ -1,33 +1,81 @@
 import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom'
 import PropTypes from 'prop-types'
 
+import { rootPath } from 'marketplace/routes'
+import ErrorBox from 'shared/form/ErrorBox'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
 import AUheading from '@gov.au/headings/lib/js/react.js'
+import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
 import BriefOverviewSection from './BriefOverviewSection'
 
 import styles from './BriefOverview.scss'
 
 export class BriefOverview extends Component {
   static propTypes = {
-    sections: PropTypes.array.isRequired
+    sections: PropTypes.array.isRequired,
+    status: PropTypes.string.isRequired
   }
 
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = {
+      showDeleteAlert: false,
+      showDeleteButton: false
```

Comment:
do you need this as internal state?  you can determine whether to display button via `this.props.status`